pkg/app/images.go

```diff
@@ -52,13 +59,22 @@ func (c *PolicyApp) Images() error {
 		if err != nil {
 			return err
 		}
-
-		arrData := []string{
-			familiarName,
-			tagOrNone,
-			descr.Digest.Encoded()[:12],
-			strings.ReplaceAll(humanize.Bytes(uint64(descr.Size)), " ", "")}
-
+		var arrData []string
+		if manifest != nil {
+			arrData = []string{
+				familiarName,
+				tagOrNone,
+				descr.Digest.Encoded()[:12],
+				manifest.Annotations[ocispec.AnnotationCreated],
+				strings.ReplaceAll(humanize.Bytes(uint64(descr.Size)), " ", "")}
+		} else {
+			arrData = []string{
+				familiarName,
+				tagOrNone,
+				descr.Digest.Encoded()[:12],
+				descr.Annotations[ocispec.AnnotationCreated],
+				strings.ReplaceAll(humanize.Bytes(uint64(descr.Size)), " ", "")}
+		}
```

Comment:
I would inverse the if and else to make it easier to read (the else is a double negation not not equal to nil => equal to nil)

```go
		if manifest == nil {
			arrData = []string{
				familiarName,
				tagOrNone,
				descr.Digest.Encoded()[:12],
				descr.Annotations[ocispec.AnnotationCreated],
				strings.ReplaceAll(humanize.Bytes(uint64(descr.Size)), " ", "")}
		} else {
			arrData = []string{
				familiarName,
				tagOrNone,
				descr.Digest.Encoded()[:12],
				manifest.Annotations[ocispec.AnnotationCreated],
				strings.ReplaceAll(humanize.Bytes(uint64(descr.Size)), " ", "")}
		}
```

Reply:
Come to thing of it I would even go a step further and refactor it to
```go
		var annotations string
		if manifest == nil {
			annotations = descr.Annotations[ocispec.AnnotationCreated]
		} else {
			annotations = manifest.Annotations[ocispec.AnnotationCreated]
		}
		
		arrData := []string{
			familiarName,
			tagOrNone,
			descr.Digest.Encoded()[:12],
			annotations,
			strings.ReplaceAll(humanize.Bytes(uint64(descr.Size)), " ", "")}
```